test/python/test_ibmqjob.py

```diff
@@ -57,6 +58,7 @@ def setUp(self, qe_token, qe_url):
         qc.measure(qr, cr)
         self._qc = qc
         self._provider = IBMQProvider(qe_token, qe_url)
+        self._local_provider = LocalProvider()
```

Comment:
`self._local_provider` is not used in any method. Can this line be removed?

Reply:
oh! you are right. It shouldn't be there.